src/Lean/Parser/Basic.lean

```diff
@@ -1610,7 +1612,8 @@ inductive LeadingIdentBehavior where
   The method `termParser prec` is equivalent to the method above.
 -/
 structure ParserCategory where
-  tables   : PrattParsingTables
+  kinds    : SyntaxNodeKindSet := {}
```

Comment:
This is change is not related to the PR.

Reply:
oops